Quotient/user.h

```diff
@@ -14,113 +14,158 @@ class Connection;
 class Room;
 class RoomMemberEvent;
 
+//! This class is designed to give interface with a given user's profile.
+//!
+//! \note The User class is not intended for getting the data to visualise a user
+//!       in the context of a particular room. For that a Quotient::RoomMember object
+//!       should be obtained from a Quotient::Room as this will account for the
+//!       user setting an avatar or name that applies to that room only.
+//!
+//! \sa Quotient::RoomMember
 class QUOTIENT_API User : public QObject {
     Q_OBJECT
     Q_PROPERTY(QString id READ id CONSTANT)
-    Q_PROPERTY(bool isGuest READ isGuest CONSTANT)
-    Q_PROPERTY(int hue READ hue CONSTANT)
-    Q_PROPERTY(qreal hueF READ hueF CONSTANT)
     Q_PROPERTY(QString name READ name NOTIFY defaultNameChanged)
     Q_PROPERTY(QString displayName READ displayname NOTIFY defaultNameChanged STORED false)
     Q_PROPERTY(QString fullName READ fullName NOTIFY defaultNameChanged STORED false)
+    Q_PROPERTY(bool isGuest READ isGuest CONSTANT)
+    Q_PROPERTY(int hue READ hue CONSTANT)
+    Q_PROPERTY(qreal hueF READ hueF CONSTANT)
     Q_PROPERTY(QString avatarMediaId READ avatarMediaId NOTIFY defaultAvatarChanged STORED false)
     Q_PROPERTY(QUrl avatarUrl READ avatarUrl NOTIFY defaultAvatarChanged)
 public:
     User(QString userId, Connection* connection);
 
     Connection* connection() const;
 
-    /** Get unique stable user id
-     * User id is generated by the server and is not changed ever.
-     */
+    //! \brief Get unique stable user id.
```

Comment:
```suggestion
    //! \brief Get unique stable user id
```
We don't use periods in `\brief` as these are, essentially, captions.